lib/vmdb/settings.rb

```diff
@@ -88,13 +88,26 @@ def self.build_template
     private_class_method :build_template
 
     def self.build(resource)
-      build_template.tap do |settings|
-        settings.add_source!(DatabaseSource.new(resource))
+      objects_in_settings_chain = resource ? resource.objects_in_settings_chain : []
+      build_without_local(objects_in_settings_chain).tap do |settings|
         local_sources.each { |s| settings.add_source!(s) } if resource.try(:is_local?)
       end
     end
     private_class_method :build
 
+    def self.parent_settings(resource)
+      objects_in_settings_chain = resource.objects_in_settings_chain
+      build_without_local(objects_in_settings_chain[0...-1]).load!
+    end
+    private_class_method :parent_settings
+
+    def self.build_without_local(objects_in_settings_chain)
```

Comment:
Since we are using the term `resource` everywhere to refer to the DatabaseSource objects, can we call this `resources_in_settings_chain` here and elsewhere?